v2/dtos/responses/deviceprofile.go

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package responses
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+// AddDeviceProfileResponse defines the Response Content for POST DeviceProfile DTOs.
+// This object and its properties correspond to the AddDeviceProfileResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/AddDeviceProfileResponse
+type AddDeviceProfileResponse struct {
+	common.BaseWithIdResponse `json:",inline"`
+}
+
+// UpdateDeviceProfileResponse defines the Response Content for PUT DeviceProfile as pushed DTO.
+// This object and its properties correspond to the UpdateDeviceProfileResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/UpdateDeviceProfileResponse
+type UpdateDeviceProfileResponse struct {
+	common.BaseWithIdResponse `json:",inline"`
+}
```

Comment:
Remove AddDeviceProfileResponse  & UpdateDeviceProfileResponse and just use common.BaseWithIdResponse. Swagger will need to be updated.